libpod/oci_conmon_linux.go

```diff
@@ -1712,6 +1644,7 @@ func httpAttachTerminalCopy(container *net.UnixConn, http *bufio.ReadWriter, cid
 				// Do nothing
 			default:
 				logrus.Errorf("Received unexpected attach type %+d, discarding %d bytes", buf[0], numR)
+				logrus.Debugf("String is %s", string(buf[1:numR]))
```

Comment:
This debug can be HUGE depending on the amount read.  That is why I deleted it the first time around.

Reply:
Good point. I'll drop this entirely

Reply:
Fixed